ui/src/variables/selectors/index.tsx

```diff
@@ -252,6 +257,9 @@ export const asAssignment = (variable: Variable): VariableAssignment => {
   }
 
   if (variable.arguments.type === 'query') {
+    if (!variable.selected[0]) {
```

Comment:
the ensures that the AST doesn't return variables if they don't have a value set, because that causes all sorts of issues in the LSP and on the api